local-helm/seleniumGrid/grid-namespace.yml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app: seleniumgrid
+  name: seleniumgrid
```

Comment:
hyphenated case for namespaces seems to be the preferred option, so selenium-grid would be better

Reply:
Next commit fixes this